### PR TITLE
Date time readme update

### PR DESCRIPTION
**Pull request by micahgodbolt**
Date-time is now officially released, but the readme didn't reflect that. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9839)

### CODE REVIEW

File: packages/date-time/README.md [hunk 1]
```diff
@@ -2,28 +2,8 @@
...
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
-
-To import DateTime components:
-
-```js
-import { ComponentName } from '@uifabric/date-time';
-```
-
-Once the DateTime component graduates to a production release, the component will be available at:
+TTo import DateTime components:
```
Comment (Vitalius1) [inline packages/date-time/README.md]:
```suggestion
To import DateTime components:
```

File: change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "updated date-time readme now that it has major released",
```
Comment (Vitalius1) [inline change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json]: nit: 😄 
```suggestion
  "comment": "Updated date-time README now that it has major released.”,
```

File: change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  "type": "patch",
```
Comment (Vitalius1) [inline change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json]: Are we patch change when updating things that don’t touch code like docs? Shouldn’t it be `none`?

Comment (micahgodbolt) [reply change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json]: Not sure if beach ball gives a none option.

Comment (Vitalius1) [reply change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json]: It does…
Screen Shot 2019-07-17 at 12 35 26

Comment (KevinTCoughlin) [reply change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json]:
```suggestion
  "type": "none",
```

Comment (micahgodbolt) [reply change/@uifabric-date-time-2019-07-17-11-47-47-date-time-readme.json]: why is none between minor and major :-p

**Pull request merged**